binder.go

```diff
@@ -48,6 +48,10 @@ func (b *binder) bindForm(ptr interface{}, form map[string][]string) error {
 	typ := reflect.TypeOf(ptr).Elem()
 	val := reflect.ValueOf(ptr).Elem()
 
+	if typ.Kind() != reflect.Struct {
```

Comment:
@o1egl You can simply return `errors.New("binding element must be a  struct")`. We are already returning `HTTPError` inside the caller.